dune-project

```diff
@@ -56,7 +56,9 @@ possible and does not make any assumptions about IO.
   ordering
   dune-build-info
   spawn
-  (odoc-parser (and (>= 2.0.0) (< 2.3.0)))
+  astring
+  result
```

Comment:
we don't need this shim. we don't support versions of OCaml that don't have the result module.